assets/content/cookbook/Beginner/string-variable-reflection.md

```diff
@@ -0,0 +1,38 @@
+[tags]: / "beginner,other"
+
+# Using string as variable (with reflection)
+
+Extract variable name from string and using it as variable
```

Comment:
I'm just using terminology defined in the Haxe Manual for the sake of consistency. See [identifier](https://haxe.org/manual/dictionary.html#identifier) dictionary entry and [`var` expression](https://haxe.org/manual/expression-var.html) manual entry for why I phrased it this way.